HomeActivity.java

```diff
@@ -0,0 +1,37 @@
+package com.example.foodrecommended_asd;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.View;
+
+import gr.net.maroulis.library.EasySplashScreen;
+
+public class HomeActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+
+        EasySplashScreen config = new EasySplashScreen(HomeActivity.this)
+                .withFullScreen()
+                .withTargetActivity(MainActivity.class)
+                .withSplashTimeOut(5000)
+                .withBackgroundColor(Color.parseColor("#1a1b29"))

```

Comment:
Consider using a color that is closer to the app's blue background, consider using #00ccff